### PR TITLE
Added a script for checking schema of generated Ansible playbooks.

### PR DESCRIPTION
**Pull request by matejak**

### CODE REVIEW

File: tests/assert_ansible_schema.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+import argparse as ap
```
Comment (jan-cerny) [inline tests/assert_ansible_schema.py]: I think you don't have to rename it.

File: tests/assert_ansible_schema.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import re
+import sys
+
+import yaml
+
+
+COMMENT_KEYS = set((
```
Comment (jan-cerny) [inline tests/assert_ansible_schema.py]: superfluous parentheses

File: tests/assert_ansible_schema.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    "platform",
+    "reboot",
+    "strategy",
+    "complexity",
+    "disruption",
+))
+
+TAGS = set((
```
Comment (jan-cerny) [inline tests/assert_ansible_schema.py]: superfluous parentheses

File: tests/assert_ansible_schema.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            .format(fname=fname, err=err)
+        )
+        print(msg, file=sys.stderr)
+        return 1
+    return 0
+
+
+if __name__ == "__main__":
+    parser = make_parser()
+    args = parser.parse_args()
```
Comment (jan-cerny) [inline tests/assert_ansible_schema.py]: I think we don't need 3 functions for parsing 1 argument.

File: tests/assert_ansible_schema.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            "A playbook tag {tag} contains @, which is unexpected.".format(tag=tag)
+
+        found = re.search(tag_regex, tag)
+        if found:
+            caught_tags.add(found.group(1))
+
+        if re.search(cce_regex, tag):
+            caught_tags.add("CCE")
+
+    assert "CCE" in caught_tags, "There is no CCE tag in the playbook"
```
Comment (jan-cerny) [inline tests/assert_ansible_schema.py]: Does every product have CCEs? Isn't it RHEL only?

File: tests/assert_ansible_schema.py (hunk 2 above)
Comment (matejak) [reply tests/assert_ansible_schema.py]: Not really, `set` accepts iterable as it's only argument. So it is a set defined using a tuple.

File: tests/assert_ansible_schema.py (hunk 1 above)
Comment (matejak) [reply tests/assert_ansible_schema.py]: OK.

File: tests/assert_ansible_schema.py (hunk 4 above)
Comment (matejak) [reply tests/assert_ansible_schema.py]: Now there are 3 arguments :-)

File: tests/assert_ansible_schema.py (hunk 5 above)
Comment (matejak) [reply tests/assert_ansible_schema.py]: You are right, moreover not all RHEL rules contain them.
As a result, this has been marked as optional.

File: cmake/SSGCommon.cmake [hunk 6]
```diff
@@ -261,6 +261,12 @@ macro(ssg_build_ansible_playbooks PRODUCT)
         NAME "${PRODUCT}-ansible-playbooks-generated-for-all-rules"
         COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/tests/ansible_playbooks_generated_for_all_rules.py" --build-dir "${CMAKE_BINARY_DIR}" --product "${PRODUCT}"
     )
+    if("${PRODUCT}" MATCHES "rhel")
+        add_test(
+            NAME "${PRODUCT}-ansible-assert-playbooks-schema"
+            COMMAND sh env "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/tests/assert_ansible_schema.py" "${CMAKE_BINARY_DIR}/${PRODUCT}/playbooks/all/*"
```
Comment (jan-cerny) [inline cmake/SSGCommon.cmake]: What is `sh env`?

File: tests/assert_ansible_schema.py (hunk 2 above)
Comment (jan-cerny) [reply tests/assert_ansible_schema.py]: I'm sorry, I meant that you should define the set normally.

File: cmake/SSGCommon.cmake (hunk 6 above)
Comment (matejak) [reply cmake/SSGCommon.cmake]: If one wants to use wildcards in commands, one needs the shell to expand wildcards.

File: tests/assert_ansible_schema.py (hunk 2 above)
Comment (matejak) [reply tests/assert_ansible_schema.py]: Clarification: Writing `my_set = {1, 2, 3}` is the best way to define a set, but it is not available on python<2.7, and we still have python2.6 in our CI.

**Comment by jan-cerny** [comment]
@matejak Thanks, but please next time add a description.

**Comment by matejak** [comment]
Sorry about that, I completely forgot.

**Pull request merged**